snuba/snapshots/bulk_load.py

```diff
@@ -14,7 +18,7 @@ class BulkLoader(ABC):
     the bulk load operation.
     """
     @abstractmethod
-    def load(self) -> None:
+    def load(self, writer: BatchWriter) -> None:
```

Comment:
Was there a reason not to just use the buffered writer here directly? It's never referenced other than when it's wrapped.

Reply:
Not a strong reason. I just did it to make the interface less restrictive since it does not logically need the writer to be buffered. I can change it.